test/test_layout_inference.cpp

```diff
@@ -0,0 +1,176 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <executor.h>
+#include <ir/all_nodes.h>
+#include <ir/builder.h>
+#include <ops/all_ops.h>
+#include <optimization/layout_inference.h>
+
+#include <test/utils.h>
+#include <test/validator.h>
+
+#include <torch/torch.h>
+
+namespace nvfuser {
+
+using testing::_;
+using testing::ElementsAre;
+
+class LayoutInferenceTest : public NVFuserTest {};
```

Comment:
```suggestion
using LayoutInferenceTest;
```

Reply:
I think there's a typo in the suggestion. But I got what you meant.